features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2091,6 +2091,24 @@ CREATE TABLE IDN_ACTION_ENDPOINT (
     FOREIGN KEY (ACTION_UUID) REFERENCES IDN_ACTION(UUID) ON DELETE CASCADE
 )
 /
+CREATE TABLE IDN_OAUTH2_TOKEN_CLAIMS (
```

Comment:
Why we don't have tenant_id?

Reply:
we use appid here which is primary key of consumer app. then we don not need tenant id here